livox_ros_driver/launch/livox_lidar.launch

```diff
@@ -10,7 +10,8 @@
 	<arg name="rviz_enable" default="false"/>
 	<arg name="rosbag_enable" default="false"/>
 	<arg name="cmdline_arg" default="$(arg bd_list)"/>
-	<arg name="msg_frame_id" default="livox_frame"/>
+	<arg name="lidar_msg_frame_id" default="livox_frame"/>
+	<arg name="imu_msg_frame_id" default="livox_frame"/>
```

Comment:
Can we use `livox_imu_link` here?

Reply:
what we could do instead of that, is pass in the argument explicitly where we call the driver, namely:
https://github.com/oxin-ros/oxin_brains/blob/main/src/lidar_algorithms/launch/livox.launch

Reply:
Sounds good. Should we also open a PR in oxin_brains modifying this file?

Reply:
yes, however this PR can be merged without changing anything in the oxin_brains repo and it should still work.